<ui version="4.0" >
 <class>WpTolerancesDialog</class>
 <widget class="QDialog" name="WpTolerancesDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>296</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>0</hsizetype>
    <vsizetype>0</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Waypoints Settings</string>
  </property>
  <widget class="QWidget" name="" >
   <property name="geometry" >
    <rect>
     <x>11</x>
     <y>11</y>
     <width>277</width>
     <height>273</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item row="1" column="1" >
     <widget class="QLabel" name="spacingLabel_2" >
      <property name="text" >
       <string>Spacing value [s] or [m/s]:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2" >
     <widget class="QLabel" name="label" >
      <property name="text" >
       <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:12pt; font-weight:600;">Waypoints Settings&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" >
     <widget class="QLabel" name="headingLabel" >
      <property name="text" >
       <string>Heading tol. (+/-)  [deg]:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>271</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" >
     <widget class="QLabel" name="spacingLabel" >
      <property name="text" >
       <string>Spacing property:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1" >
     <widget class="QSpinBox" name="headingSpin" >
      <property name="maximum" >
       <number>180</number>
      </property>
      <property name="singleStep" >
       <number>5</number>
      </property>
      <property name="value" >
       <number>90</number>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="2" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>271</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="8" column="1" >
     <widget class="QSpinBox" name="maxTurnrateSpin" >
      <property name="maximum" >
       <number>2000000</number>
      </property>
      <property name="singleStep" >
       <number>1</number>
      </property>
      <property name="value" >
       <number>5</number>
      </property>
     </widget>
    </item>
    <item row="4" column="0" >
     <widget class="QLabel" name="distanceLabel" >
      <property name="text" >
       <string>Distance tolerance [m]:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" >
     <widget class="QComboBox" name="spacingPropertyCombo" >
      <item>
       <property name="text" >
        <string>Velocity</string>
       </property>
      </item>
      <item>
       <property name="text" >
        <string>Time</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="8" column="0" >
     <widget class="QDoubleSpinBox" name="maxSpeedSpin" >
      <property name="maximum" >
       <double>2000000.000000000000000</double>
      </property>
      <property name="singleStep" >
       <double>0.200000000000000</double>
      </property>
      <property name="value" >
       <double>0.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="1" >
     <widget class="QDoubleSpinBox" name="spacingValueSpin" >
      <property name="decimals" >
       <number>1</number>
      </property>
      <property name="maximum" >
       <double>10000.000000000000000</double>
      </property>
      <property name="singleStep" >
       <double>0.100000000000000</double>
      </property>
      <property name="value" >
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="2" >
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>111</width>
          <height>31</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="okButton" >
        <property name="text" >
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton" >
        <property name="text" >
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="1" >
     <widget class="QLabel" name="maxTurnrateLabel" >
      <property name="text" >
       <string>Maximum turnrate [deg/s]:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" >
     <widget class="QDoubleSpinBox" name="distanceSpin" >
      <property name="maximum" >
       <double>10.000000000000000</double>
      </property>
      <property name="singleStep" >
       <double>0.200000000000000</double>
      </property>
      <property name="value" >
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>271</width>
        <height>16</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="0" >
     <widget class="QLabel" name="maxSpeedLabel" >
      <property name="text" >
       <string>Maximum speed [m/s]:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>WpTolerancesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>WpTolerancesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
